metadatastore/examples/sample_data/common.py

```diff
@@ -69,28 +69,3 @@ def noisy(val, sigma=0.01):
         return val + sigma * np.random.randn()
     else:
         return val + sigma * np.random.randn(len(val)).reshape(val.shape)
-
-
-get_time = ttime.time
-
-
-def example(func):
```

Comment:
This was overbuilt (by me) in the first place, and I'm removing it now because it makes it impossible to inject which MDS instance to use at runtime.